src/configure

```diff
@@ -1244,6 +1244,15 @@ elif [ "`uname`" == "Linux" ]; then
       echo MKLLIB = $MKLLIBDIR >> kaldi.mk
     fi
     echo >> kaldi.mk
+
+    echo "MKL_LD_PRELOAD := $MKLROOT/lib/intel64/libmkl_def.so" >> kaldi.mk
+    echo "MKL_LD_PRELOAD += $MKLROOT/lib/intel64/libmkl_avx2.so" >> kaldi.mk
+    echo "MKL_LD_PRELAOD += $MKLROOT/lib/intel64/libmkl_core.so" >> kaldi.mk
```

Comment:
Typo: PRELAOD.

I don't think you needed to worry about the indentation.  It's OK if not all lines are  properly indented.   But if spaces work fine, that's OK with me.

Reply:
thanks, I'll fix the typo. It seems that not all `*.so` are needed since I did NOT include all
of them because of the typo.

Reply:
fixed.